client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -32,10 +35,17 @@ class Transfer extends React.PureComponent {
 			content = this.getCancelledContent();
 		}
 
+		const path = cancelPurchaseLink( this.props.selectedSite.slug, domain.subscriptionId );
+
 		return (
 			<div className="edit__domain-details-card">
 				<Header domain={ domain } />
 				{ content }
+				<VerticalNav>
+					<VerticalNavItem path={ path }>
+						{ this.props.translate( 'Cancel Domain Transfer' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Cancel Transfer' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).